src/constants/indicators.js

```diff
@@ -431,15 +431,15 @@ export default {
   newBusinessEntrants: {
     name: 'New business entry rate',
     legendText: 'Entrants',
-    format: FORMATS.THREE_DECIMALS,
+    format: FORMATS.PERCENT_ZERO_OR_ONE_DECIMAL,
```

Comment:
A question for @jamesbroadbent202 - if this is a percent, should the description be changed from "...as a share of the total..." to "...as a percentage of the total..."?

Reply:
Have changed to "... as a percentage of the stock" for entries & exits.